app/helpers/info_request_helper.rb

```diff
@@ -269,7 +269,7 @@ def attachment_path(attachment, options = {})
   end
 
   def attachment_url(attachment, options = {})
-    attach_params = attachment_params(attachment, options)
+    attach_params = attachment_params(attachment, options).merge(locale: false)
```

Comment:
So one Downside of this is that we lose the translations of the view prefix:

**No locale**

![Screenshot 2020-04-20 at 15 45 01](https://user-images.githubusercontent.com/282788/79764979-f53e1c00-831d-11ea-95d0-84f6c7804441.png)


**With Locale**

![Screenshot 2020-04-20 at 15 44 22](https://user-images.githubusercontent.com/282788/79764914-e2c3e280-831d-11ea-963d-14b2508855f1.png)

Reply:
I think what we ideally want to do is keep the locale-based views, cache all locales, but only cache the extracted attachments once, rather than only cache this view once.

We should drop this commit.